ansible_collections/arista/avd/roles/eos_designs/docs/tables/node-type-structure.md

```diff
@@ -24,6 +28,11 @@
       # Define variables for all nodes of this type.
       defaults:
 
+        # ssl_profile_name will be used as a name for cert,
+        # keys which will be internally created by CVP and pushed to respective devices.
+        # Used for communication with stun server.
+        ssl_profile: <str>
```

Comment:
IMO we don't need to set this here. Having it under the wan_route_servers model should be enough.

Reply:
The way I take wan_route_servers is a place holder to specify the pathfinder details only if it is not part of the same inventory. Else just the name is enough. 
I feel slowly the things may grow up in such a way that some options will be under wan router servers and some under the node definition. 

How about keeping at global level in wan_route_servers and thus one ssl_profile for all pathfinders?
( As we had the cluster concept in studios and ssl_profile was part of the cluster section. )

I am ok to do it as per your suggestion as well but in case my points sounds good :)

Reply:
I would have a default SSL profile name, so this setting would only be used if you wish to use a non-default setting. I would expect most deployments to just accept the default.